### PR TITLE
[RW-10031] Remove deprecated Google auth API usage

### PR DESCRIPTION
**Pull request by Peter-Lavigne**
Duplicates https://github.com/DataBiosphere/terra-docker/pull/421, see [this comment](https://github.com/DataBiosphere/terra-docker/pull/421#issuecomment-1654414371).

### CODE REVIEW

**Comment by Peter-Lavigne** [comment]
For each failing image (bioconductor, hail, and python), I have manually built the images and followed the manual testing steps described in [this commit](https://github.com/DataBiosphere/terra-docker/pull/446/commits/5eeafe8f8ed93b06fc1300dbd9835800ea865818). As described in [this message](https://broadinstitute.slack.com/archives/C046XJD9MPW/p1690571245729819?thread_ts=1690485376.690809&cid=C046XJD9MPW), I should be OK to merge this with these checks failing.

However, there is a new merge conflict in three CHANGELOG files, so I will rebase and re-run the version updating script.

File: terra-jupyter-hail/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.1.2 - 2023-08-01T15:57:27.329878Z
+
+- Update `terra-jupyter-base` to `1.1.2`
```
Comment (jmthibault79) [inline terra-jupyter-hail/CHANGELOG.md:3]: should we include all the dependency bumps here?  Specifically terra-jupyter-python in this case.

**Comment by Peter-Lavigne** [comment]
As described [here](https://precisionmedicineinitiative.atlassian.net/browse/RW-10031?focusedCommentId=170876) and in much less detail [here](https://broadinstitute.slack.com/archives/C046XJD9MPW/p1690905797909769?thread_ts=1690485376.690809&cid=C046XJD9MPW), I'm closing this PR.

File: terra-jupyter-hail/CHANGELOG.md (hunk 1 above)
Comment (Peter-Lavigne) [reply terra-jupyter-hail/CHANGELOG.md:3]: I [asked](https://broadinstitute.slack.com/archives/C046XJD9MPW/p1690485376690809) about updating the base image only but never got a response to that specific suggestion.

**Pull request closed without merging**